test/llvm/CMakeLists.txt

```diff
@@ -25,7 +25,11 @@
  ###############################################################################
 
 # Find / configure LLVM
-find_package(LLVM REQUIRED CONFIG)
```

Comment:
1. Replaced REQUIRED with QUIET to print custom error message
2. NO_DEFAULT_PATH + HINTS => only searching LLVM in `HINTS`